mobify/platform-scaffold
# Add support for Samsung native browser

## Changes
- Add babel polyfill
- Fix a bug that was breaking the PDP on Samsung

## How to test-drive this PR
- Run `npm i`
- Run `npm run dev`
- Preview [Merlin's Potions](https://preview.mobify.com/?url=https%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1) in the Samsung native browser
- smoke test the site and make sure everything works

## Note
Checkout appears to be broken on develop and this branch. If you refresh the page while you're at `https://www.merlinspotions.com/checkout/payment/`, `extractMagentoShippingStepData` will fail
